src/Header/AbstractRecipient.php

```diff
@@ -57,4 +57,26 @@
             ])
         );
     }
+
+    /**
+     * @param array $list pairs of email address and name
+     * @return AbstractRecipient
+     */
+    final public static function fromList(array $list): AbstractRecipient
+    {
```

Comment:
Thought about the name `fromList`. Since we already have an `AddressList`, I rather call this `fromArray`.